api/data_explorer/__main__.py

```diff
@@ -268,24 +259,31 @@ def _get_es_facets():
             # is fixed, use AutoHistogramFacet instead. Will no longer need 2
             # steps.
             max_field_value = _get_max_field_value(es, field_name)
-            facets[ui_facet_name] = HistogramFacet(
+            es_facets[ui_facet_name] = HistogramFacet(
                 field=field_name, interval=_get_interval(max_field_value))
-    app.app.logger.info('Elasticsearch facets: %s' % facets)
-    return facets
+
+    app.app.logger.info('Elasticsearch facets: %s' % es_facets)
+    app.app.config['ELASTICSEARCH_FACETS'] = es_facets
+    app.app.logger.info('UI facets: %s' % ui_facets)
+    app.app.config['UI_FACETS'] = ui_facets
 
 
-def _get_table_names():
+def _process_bigquery_config():
     """Gets an alphabetically ordered list of table names from bigquery.json.
     Table names are fully qualified: <project id>.<dataset id>.<table name>
     If bigquery.json doesn't exist, this returns an empty list.
     """
     config_path = os.path.join(app.app.config['DATASET_CONFIG_DIR'],
                                'bigquery.json')
     table_names = []
+    primary_key = ""
     if os.path.isfile(config_path):
-        table_names = _parse_json_file(config_path)['table_names']
+        bigquery_data = _parse_json_file(config_path)
```

Comment:
nit we usually call this `bigquery_config`, it's a bit more specific than bigquery_data

Reply:
Done